as_perimenoun_oi_gynaikes.ass

```diff
@@ -313,20 +313,21 @@ Dialogue: 0,0:20:02.84,0:20:06.30,Default,,0,0,0,,When you saw the girl and yell
 Dialogue: 0,0:20:06.30,0:20:08.94,Default,,0,0,0,,- As if it was her fault.\N- She caused your destabilization.
 Dialogue: 0,0:20:08.94,0:20:10.50,Default,,0,0,0,,Get to your senses!
 Dialogue: 0,0:20:10.50,0:20:11.98,Default,,0,0,0,,It was he who destabilized me!
-Dialogue: 0,0:20:11.98,0:20:16.04,Default,,0,0,0,,Because he wants to trap us. So that we take care of the \Nseniors while he goes spearfishing.
-Dialogue: 0,0:20:16.04,0:20:21.18,Default,,0,0,0,,- Indeed, he might have thought of that.\N- Yes, but what he has not thought of\N is that I might get crazy...
-Dialogue: 0,0:20:21.18,0:20:23.58,Default,,0,0,0,,...and throttle his mommy at the central square.
+Dialogue: 0,0:20:11.98,0:20:15.62,Default,,0,0,0,,Because he wants to trap us. So that we take care of the \Nseniors while he goes spearfishing.
+Dialogue: 0,0:20:15.70,0:20:16.98,Default,,0,0,0,,- Indeed, he might have thought of that.
+Dialogue: 0,0:20:17.10,0:20:21.82,Default,,0,0,0,,Yes, but what he hasn't thought that I might get crazy and throttle his mommy at the central square.
+Dialogue: 0,0:20:21.94,0:20:29.16,Default,,0,0,0,,[SONG] {\i1}Don't tell lies to me, like those you tell to many others, because I might believe them{\i0}
 Dialogue: 0,0:20:29.65,0:20:36.09,Default,,0,0,0,,- Is that radio?\N - CD.\N - Again, good day to you.\N
 Dialogue: 0,0:20:46.48,0:20:48.68,Default,,0,0,0,,Panos, leave Antonis to God.
 Dialogue: 0,0:20:48.68,0:20:50.68,Default,,0,0,0,,We need to get back to our families.
-Dialogue: 0,0:20:50.68,0:20:53.25,Default,,0,0,0,,He will sail and he will ask the strangers...
+Dialogue: 0,0:20:50.68,0:20:53.25,Default,,0,0,0,,He will go to the ferry and he will ask the strangers...

```

Comment:
No ferry is refereed here. What they mean is that he will sail away ("μπαρκάρω"). This is contextually wrong.

Reply:
Nope. https://twitter.com/apogpanos/status/906888887489974272
Θα πηγαίνει στα καράβια και θα ρωτάει τον άγνωστο κόσμο
“Μήπως είδατε τα μπατζανάκια μου”;
Antonis, will not become sailor and he will not travel. He will go the ferries/ships at the dock to ask the whereabouts for his brothers in law.

Reply:
Yeah, you are right. I would prefer then if it would be "He will go down to the docks" and not "to the ferry".